app/scripts/controllers/network/network-controller.js

```diff
@@ -22,14 +23,20 @@ import {
   CHAIN_IDS,
   NETWORK_TYPES,
 } from '../../../../shared/constants/network';
-import {
-  isPrefixedFormattedHexString,
-  isSafeChainId,
-} from '../../../../shared/modules/network.utils';
 import getFetchWithTimeout from '../../../../shared/modules/fetch-with-timeout';
 import createInfuraClient from './createInfuraClient';
 import createJsonRpcClient from './createJsonRpcClient';
 
+/**
+ * @typedef {object} NetworkConfiguration
+ * @property {string} rpcUrl - RPC target URL.
+ * @property {string} chainId - Network ID as per EIP-155
+ * @property {string} chainName - Personalized network name.
```

Comment:
`chainName` and `ticker` used to be optional in the preferences controller. I see that they're required by `addEthereumChain` but it's less clear to me whether our network form requires it.

It looks to me like our network form currently requires `ticker`, but the nickname/chainName is less clear to me.

Reply:
You're right. We do require a ticker now, but not a chainName. Will mark this as optional

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/17421/commits/6dbe2ac83e0e2890d7670339979f8bc4e005e4e4